### PR TITLE
Release 5.0.0

### PR DESCRIPTION
**Pull request by winged**
Note: I've added documentation on HOWTO release, please verify that as well

### CODE REVIEW

File: docs/maintainers.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Release Management
+
+The release procedure was recently changed from automatic to manual, due to too
+many problems with it.
+
+Here is a rough outline / checklist for the release:
+
+1. Checkout `master` branch, ensure you have all tags
+2. Prepare changelog
```
Comment (czosel) [inline docs/maintainers.md]: Can you add the command that generates release notes?

File: docs/maintainers.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+3. Figure out the next version
+4. Update verion in the code
```
Comment (czosel) [inline docs/maintainers.md]:
```suggestion
4. Update version in the code
```

File: docs/maintainers.md (hunk 1 above)
Comment (czosel) [reply docs/maintainers.md]: Ah nevermind, this follows below.

Comment (czosel) [reply docs/maintainers.md]:
```suggestion
2. Prepare changelog (see below)
```

File: docs/maintainers.md (hunk 2 above)
Comment (czosel) [reply docs/maintainers.md]: Same as above, mentioned below.

File: docs/maintainers.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+### Update version in source
+
+The version is also put in code. Update the file
+`caluma/caluma_metadata.py`.
+
+### Create a Pull request for the proposed version bump
+
+Put the changelog in the commit message or in the PR discussion somewhere, so
+it won't be forgotten once the release actually happens.
```
Comment (czosel) [inline docs/maintainers.md]: Should we consider using `CHANGELOG.md`?

Comment (winged) [reply docs/maintainers.md]: I like the idea!

Comment (winged) [reply docs/maintainers.md]: Added CHANGELOG with historical entries

**Comment by winged** [comment]
Thanks a lot for your feedback @czosel . Can you have a look again?

**Pull request merged**